.travis.yml

```diff
@@ -37,6 +37,7 @@ before_install:
        --allow-external petsc --allow-unverified petsc \
        --allow-external petsc4py  --allow-unverified petsc4py \
        < requirements-git.txt"
+  - pip install pulp
```

Comment:
I only need this so that the pyop2 tests pass on travis (i.e. coffee is importable).